Source/Scene/Fog.js

```diff
@@ -114,7 +114,7 @@ define([
         var positionCartographic = camera.positionCartographic;
 
         // Turn off fog in space.
-        if (!defined(positionCartographic) || positionCartographic.height > 800000.0 || frameState.mode !== SceneMode.SCENE3D) {
+        if (!defined(positionCartographic) || positionCartographic.height > 800000.0 || frameState.mode !== SceneMode.SCENE3D || frameState.terrainExaggeration > 1.0) {
```

Comment:
What does fog have to be disabled when exaggeration is greater than one?

What about less than one?